d3a_api_client/utils.py

```diff
@@ -197,6 +197,7 @@ def list_running_canary_networks_and_devices_with_live_data(domain_name):
     data = endpoint(query=query)
 
     return {
-        cn["uuid"]: json.loads(cn["scenarioData"]["forecastStreamAreaMapping"])
+        cn["uuid"]: cn["scenarioData"]["forecastStreamAreaMapping"]
         for cn in data["data"]["listCanaryNetworks"]["configurations"]
-    }
+        if cn["resultsStatus"] == "running"
```

Comment:
`else None` is not needed here?

Reply:
No, if the Canary Network is not running then we simply do not connect to it when the MQTT subscriber starts. When the CN starts the MQTT subscriber will be notified via the Redis queue.